### ISSUE TITLE
Release comment are missing on PRs

### ISSUE THREAD

**Issue by stefanbuck** [release-notifier/release-notifier#10]
Yesterday I install the release-notifier app on the OctoLinker repo. Today I've created a new release and I expected to see comments on all past PRs between this and the previous release.

For some reason I only got one comment on the version bump PR itself https://github.com/OctoLinker/OctoLinker/pull/439. All the referenced PRs doesn't have a comment. 

I browsed through the release-`notifier` source, but I couldn't find anything. Let me know how I can help to track it down.

**Comment by gr2m** [comment]
I can confirm that something is wonky. We’ll have to wait for @zeke to come back, I don’t have access to the logs

**Comment by zeke** [comment]
@stefanbuck sorry about that. I've added some logging in #7 but that was after your release.

@gr2m what is your Heroku email address? I'll add you as a collaborator on the app.

**Comment by yoshuawuyts** [comment]
Similar effect on https://github.com/choojs/create-choo-app/pull/45. Went back in here to check in now that logs have been enabled.

**Comment by zeke** [comment]
It seems that some webhooks are not being received by the app. For example, I can search the logs for `create-choo-app`, and I see numerous events like this over the last few days:

```
17:18:08.044Z  INFO probot: Webhook received 
  event: { 
    "id": "88b04400-0a98-11e8-973b-484ed3cc3349", 
    "event": "issues.opened", 
    "repository": "choojs/create-choo-app", 
    "installation": 84031 
  } 
```

(Note that I have not instrumented the app to log this specific info ☝️. Probot logs this stuff by default.)

However none of these events are of type `release.published`, even though a release of `create-choo-app` was indeed created in the last day.

**Comment by zeke** [comment]
In other news, the new logging setup is successfully exposing more obvious bugs for some other repos, like this one: https://github.com/release-notifier/release-notifier/issues/12

**Comment by bkeepers** [comment]
> However none of these events are of type `release.published`, even though a release of create-choo-app was indeed created in the last day.

Does the GitHub App settings have the `release` even enabled? For example, when creating a [new app](https://github.com/settings/apps/new), you have to select **Read-only** on **Repository contents**, and then select the **Releases** event.

**Comment by gr2m** [comment]
the release event is selected

<img width="383" alt="image" src="https://user-images.githubusercontent.com/39992/35874973-cebef6b4-0b22-11e8-8417-4a183e615ef7.png">

**Comment by zeke** [comment]
Oh yeah sorry. Forgot to mention that the `release.published` event is being logged for some repos.:

```
Feb 01 13:21:28 release-notifier app/web.1:     "event": "release.published", 
Feb 01 21:32:36 release-notifier app/web.1:     "event": "release.published", 
Feb 02 09:32:18 release-notifier app/web.1:     "event": "release.published", 
Feb 02 12:39:57 release-notifier app/web.1:     "event": "release.published", 
Feb 02 20:04:09 release-notifier app/web.1:     "event": "release.published", 
Feb 03 06:52:07 release-notifier app/web.1:     "event": "release.published", 
Feb 03 13:41:47 release-notifier app/web.1:     "event": "release.published", 
Feb 04 12:48:56 release-notifier app/web.1:     "event": "release.published", 
Feb 06 14:04:53 release-notifier app/web.1:      "event": "release.published", 
```

**Comment by yoshuawuyts** [comment]
@zeke could it be that "release.published" doesn't actually refer to a release, but rather to a tag? I remember queueing GitHub at some point, and GitHub considers the two to be different.

With GraphQL I had to do something like this:
```js

var projectTags = gql`
  query projectTags ($owner: String!, $name: String!) {
    repository(owner: $owner, name: $name) {
      tags:refs(refPrefix:"refs/tags/", last:2) {
        edges {
          tag:node {
            name
            target {
              commitResourcePath
            }
          }
        }
      }
    }
  }
`
```
Not sure what y'all doing in the code, but this might perhaps be helpful?

**Comment by zeke** [comment]
Good point, @yoshuawuyts. Watching for tag events might make more sense.
